data/DAC/metadata.json

```diff
@@ -1,5 +1,30 @@
 {
-  "contributors": [],
+  "contributors": [
+    {
+      "Type": "cdr company",
```

Comment:
The uppercased **values** were fine, but it'd be great to normalize the **keys** to be lowercased!

Reply:
Oh haha, good catch! Will revert and update.